dev/io.openliberty.faces.internal/bnd.bnd

```diff
@@ -41,16 +39,16 @@ Service-Component: \
     greedy:="cdiJSFInitializerService"; \
     optional:="cdiJSFInitializerService"; \
     properties:="service.vendor=IBM", \
-  com.ibm.ws.myfaces.initializer; \
+  io.openliberty.myfaces.initializer; \
     implementation:=com.ibm.ws.jsf.ee.WASMyFacesContainerInitializer; \
     provide:=javax.servlet.ServletContainerInitializer; \
     properties:="service.vendor=IBM,service.ranking:Integer=101", \
-  com.ibm.ws.jsf_3_0.injection.class.list.collaborator; \
+  io.openliberty.faces.injection.class.list.collaborator; \
     implementation:=com.ibm.ws.jsf.shared.util.JSFInjectionClassListCollaborator ; \
     provide:='com.ibm.wsspi.webcontainer.collaborator.WebAppInjectionClassListCollaborator'; \
     immediate:=true; \
     properties:="service.vendor=IBM", \
-  com.ibm.ws.org.apache.myfaces_3_0_${replace;${bVersion};\\.;_}.ResourceProvider; \
+  io.openliberty.org.apache.myfaces_${replace;${bVersion};\\.;_}.ResourceProvider; \
```

Comment:
changing files names to be more generic since this bundle applies to jsf 23 & faces 30

Reply:
Makes sense since this is shared! Thanks!